main.go

```diff
@@ -533,16 +359,21 @@ This means that when the API changes
 		"deliver",
 	}
 
-	if cfg.ItunesConnectUser != "" {
-		args = append(args, "--username", cfg.ItunesConnectUser)
+	if authConfig.AppleID != nil && authConfig.AppleID.username != "" {
+		args = append(args, "--username", authConfig.AppleID.username)
```

Comment:
why is not this in the block starting at line 340? (`if authConfig.AppleID != nil {`)

Reply:
i see: since passwords and session are passed as env, while apple id as arg.

might it worth to keep auth related configuration close to each other

Reply:
Moved to helper function (AppendFastlaneCredentials)